### PR TITLE
Events ipc command names fix

### PR DESCRIPTION
**Pull request by mary-dcouto**

### CODE REVIEW

File: lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h [hunk 1]
```diff
@@ -86,39 +87,39 @@ XX(identifier, string, none, id, __VA_ARGS__) \
 XX(dump_path, string, none, dump_path, __VA_ARGS__)
 
 #define TNL_ENABLE_MFA(XX, ...) \
-XX(identifier, string, none, id, __VA_ARGS__)
+XX(identifier, string, none, Identifier, __VA_ARGS__)
```
Comment (scareything) [inline lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:90]: Is there a reason to capitalize the json fields? It seems tedious to me.

Comment (scareything) [reply lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:90]: I see. You want to camel-case the fields that are compound words, which is better than using underscores.

File: programs/ziti-edge-tunnel/netif_driver/windows/tun.c [hunk 2]
```diff
@@ -226,7 +224,7 @@ netif_driver tun_open(struct uv_loop_s *loop, uint32_t tun_ip, uint32_t dns_ip,
     return driver;
 }
 
-static int tun_close(struct netif_handle_s *tun) {
```
Comment (scareything) [inline programs/ziti-edge-tunnel/netif_driver/windows/tun.c:229]: `tun_close` should not be exposed and called directly.

File: lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h (hunk 1 above)
Comment (mary-dcouto) [reply lib/ziti-tunnel-cbs/include/ziti/ziti_tunnel_cbs.h:90]: I made those field names same as what UI currently supports, WDE UI is using CamelCase

File: programs/ziti-edge-tunnel/ziti-edge-tunnel.c [hunk 3]
```diff
@@ -2252,6 +2280,14 @@ void scm_service_stop() {
 
     remove_all_nrpt_rules();
 
+    // tun close needs to be fixed
```
Comment (scareything) [inline programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2283]: what's wrong with tun_close? can we fix it now in this PR?  If not, we should create an issue in GH and maybe make this comment a TODO that references the issue.

Comment (mary-dcouto) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2283]: When we call tun_close, the ziti_loop thread is crashing. It looks like it is not killing the sessions gracefully. I will file an issue against this repo for this error.

Comment (mary-dcouto) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2283]: https://github.com/openziti/ziti-tunnel-sdk-c/issues/275

Comment (scareything) [reply programs/ziti-edge-tunnel/ziti-edge-tunnel.c:2283]: please add the GitHub issue link as a comment, so it survives this PR.

**Review by scareything — APPROVED** [review]

**Pull request merged**